src/Symfony/Component/Security/Acl/Dbal/AclProvider.php

```diff
@@ -192,7 +193,8 @@ public function findAcls(array $oids, array $sids = array())
                         $this->cache->putInCache($loadedAcl);
                     }
 
-                    if (isset($oidLookup[$loadedOid->getIdentifier().$loadedOid->getType()])) {
+                    $strOidLookupKey = $loadedOid->getIdentifier() . $loadedOid->getType();
```

Comment:
By the way, what is the reason for this change at all? The resulting string is only used once.